File: Week-08/Md Sohel/Red Light, Green Light.cpp
#include <iostream>
using namespace std;

int main() {
	// your code goes here
	int t, n , k, shot, p;
	
	cin >> t;
	while(t--){
	    shot = 0;
	    cin >> n;
	    cin >> k;
	    while(n--){
	        cin >> p;
	        if(p > k){
	            shot++;
	        }
	    }
	    cout << shot << "\n";
	}
	return 0;
}

File: Week-08/Sabiha Moon/Red Light, Green Light.java
/* package codechef; // don't place package name! */

import java.util.*;
import java.lang.*;
import java.io.*;

/* Name of the class has to be "Main" only if the class is public. */
class Codechef
{
	public static void main (String[] args) throws java.lang.Exception
	{
		// your code goes here
		Scanner sc= new Scanner(System.in) ;
		int T = sc.nextInt() ;
		while(T!=0){
		    int n = sc.nextInt() ;
		     int [] arr = new int[n] ;
		     int m = sc.nextInt() ;
		    for (int i = 0 ; i<n ;i++){
		        arr[i] = sc.nextInt() ;
		    }
		    int k =0 ;
		    for (int i = 0 ; i<n ;i++){
		        if(arr[i]>m){
		            k++ ;
		        }
		    }
		    System.out.println(k) ;
		    T-- ;
		}
		
		
	}
}
